### ISSUE TITLE
Select to next/previous empy line

### ISSUE THREAD

**Issue by khvorov45** [slynch8/10x#1799]
10x has the commands to move the cursor to the next/previous empty line:

MoveToNextEmptyLine
MoveToPreviousEmptyLine

But there is there a way to move and select?

**Comment by slynch8** [comment]
no, but you could probably write a simple python function to do this

**Comment by khvorov45** [comment]
I did for now

```py
import N10X

def MoveToPreviousEmptyLineSelect():
    curSelection = N10X.Editor.GetCursorSelection()
    posBefore = curSelection[1]
    N10X.Editor.ExecuteCommand("MoveToPreviousEmptyLine")
    posAfter = N10X.Editor.GetCursorPos()
    N10X.Editor.SetSelection(posBefore, posAfter)

def MoveToNextEmptyLineSelect():
    curSelection = N10X.Editor.GetCursorSelection()
    posBefore = curSelection[0]
    N10X.Editor.ExecuteCommand("MoveToNextEmptyLine")
    posAfter = N10X.Editor.GetCursorPos()
    N10X.Editor.SetSelection(posBefore, posAfter)
```

Except that 10x crashes sometimes when I do execute those. Or maybe it crashes for a different reason, I don't know. I uploaded the crash dumps.

**Comment by slynch8** [comment]
You functions look good so I don't think they are causing the crash. Looking at the crash dumps it's crashing in the parser. Is it crashing on a project you can share?

**Comment by khvorov45** [comment]
Sure
https://github.com/khvorov45/triaxis

I also added the windows sdk directory to the workspace though
Windows Kits\10\Include\10.0.18362.0

**Comment by slynch8** [comment]
Thanks. Could you also share your .10x file, I'd like to see how you've set it up

**Comment by khvorov45** [comment]
Attached. Renamed to .txt because github won't upload otherwise.

[triaxis.10x.txt](https://github.com/slynch8/10x/files/11717061/triaxis.10x.txt)

**Comment by slynch8** [comment]
Thanks. That all looks fine. Unfortunately it doesn't repro the crash. Let me know if you get it again.
